File: app/src/main/java/com/example/myapplication/MyDataModel.java
package com.example.myapplication;

public class MyDataModel {


    String firstname,lastname,gr,id;

    public MyDataModel(){

    }

    public MyDataModel(String id, String firstname, String lastname, String gr){
        this.id=id;
        this.firstname=firstname;
        this.lastname=lastname;
        this.gr=gr;

    }
    public  String getId(){
        return id;
    }

    public String getGr() {
        return gr;
    }

    public String getFirstname() {
        return firstname;
    }

    public String getLastname() {
        return lastname;
    }
    public int size(){
        return 28;
    }
}

File: app/src/main/java/com/example/myapplication/MyDataModel.kt
package com.example.myapplication

class MyDataModel(id: String?, firstname: String?, lastname: String?, gr: String?) {
    //Реализация модели наших данных для использования в дальнейшем
    var firstname: String? = firstname
        private set
    var lastname: String? = lastname
        private set
    var gr: String? = gr
        private set
    var id: String? = id
        private set

}